notebooks/ArynPartitionerWithLangchain.ipynb

```diff
@@ -112,7 +119,7 @@
     "\n",
     "\n",
     "partitioned_docset = pdf_docset.partition(\n",
-    "    partitioner=ArynPartitioner(threshold=0.35, extract_table_structure=True, aryn_api_key = aryn_api_key, use_partitioning_service = True ) \n",
+    "    partitioner=ArynPartitioner(threshold=0.35, extract_table_structure=True, aryn_api_key = aryn_api_key) \n",
```

Comment:
If users do the env var or put the file as ~/.aryn/config.yaml, the aryn_api_key = aryn_api_key should be unneeded.

Reply:
thanks Eric, I will update the notebook.